libs/full/threadmanager/src/threadmanager.cpp

```diff
@@ -105,6 +82,7 @@ namespace hpx { namespace threads {
         notifier.add_on_stop_thread_callback(
             util::bind(&threadmanager::deinit_tss, this));
 
+        // TODO: Pass reference to partitioner through constructor.
```

Comment:
Should it be removed? Is this for another PR?

Reply:
Removed, thanks! This is potentially for a future PR.